tsconfig.json

```diff
@@ -1,29 +1,7 @@
 {
+  "extends": "@waveshq/standard-web/tsconfig.json",
   "compilerOptions": {
-    "target": "es5",
-    "lib": ["dom", "dom.iterable", "esnext"],
-    "allowJs": true,
-    "skipLibCheck": true,
-    "strict": false,
-    "strictNullChecks": true,
-    "forceConsistentCasingInFileNames": true,
-    "noEmit": true,
-    "esModuleInterop": true,
-    "module": "esnext",
-    "moduleResolution": "node",
-    "resolveJsonModule": true,
-    "isolatedModules": true,
-    "jsx": "preserve",
-    "incremental": true,
-    "baseUrl": "src",
-    "paths": {
-      "@api/*": ["api/*"],
-      "@content/*": ["../content/*"],
-      "@contexts/*": ["layouts/contexts/*"],
-      "@components/*": ["components/*"],
-      "@store/*": ["store/*"],
-      "@public/*": ["../public/*"]
-    }
+    "baseUrl": "src"
```

Comment:
i saw this is in `standard-web`, do we need to define it again?
ref: https://github.com/WavesHQ/standard/blob/72b912ad9a8563a233860af9e0de09c30272af75/packages/standard-web/tsconfig.json#L20

Reply:
Yes because it's relative to the current dir